examples/realtime/lslclient_rt_plot.py

```diff
@@ -0,0 +1,48 @@
+"""
+==============================================================
+Plot real-time epoch data with LSL client
```

Comment:
We should eventually aim to have the realtime examples run with CircleCI. Otherwise it becomes hard to maintain.

Reply:
what do you mean by this?

Reply:
The example should start with `plot_` so if someone makes a change that breaks the example, CircleCI should fail. Same with all the other realtime examples.

Reply:
@jasmainak gotcha, wanna create an issue for that?

Reply:
I added it as a todo here: https://github.com/mne-tools/mne-python/issues/6157#issue-433854527